providers/pipedrive/read.go

```diff
@@ -0,0 +1,53 @@
+package pipedrive
+
+import (
+	"context"
+	"time"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+)
+
+func (c *Connector) Read(ctx context.Context, config common.ReadParams) (*common.ReadResult, error) {
+	if err := config.ValidateParams(true); err != nil {
+		return nil, err
+	}
+
+	url, err := c.buildReadURL(config)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := c.Client.Get(ctx, url.String())
+	if err != nil {
+		return nil, err
+	}
+
+	return common.ParseResult(resp,
+		getRecords,
```

Comment:
This method can be replaced with general method for get records.
```go
common.GetRecordsUnderJSONPath("data")
```
It expects an array of objects under key field "data".